packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/templates/expandableSearchBoxInput.css

```diff
@@ -25,7 +25,7 @@
 }
 
 .z-index-high {
-  z-index: 20;
+  z-index: 5;
```

Comment:
If it was put at 20 it's probably because there were some other things that this was showing up under...

Are you certain this isn't making the input appear under something else? for example in the standalone searchbox?

Reply:
the 20 was set by me just so that it makes 20s increments in this [PR](https://github.com/coveo/ui-kit/pull/3455) when we fixed the tooltip which was overflowing and being partly hidden.

Before that you set 30 in this [PR](https://github.com/coveo/ui-kit/pull/3144), there is no indication why it was set to 30 maybe you remember? 

It works well with quantic summary and quantic Sort.

For RGA only the tooltips had z-index of 40 so that clears. I could change the 40 to something that makes more sense maybe 10 and do increments of 5?

But because of the `cSearchPublisher ` as long as its above 10 we'll have the issue

Reply:
as discussed, the original 30 was to clear the default template and make sure the HSSB is always on top, we tested and it works fine with a z-index of 5

Reply:
👍